content/client-lib-development-guide/features.textile

```diff
@@ -482,12 +482,12 @@ h3(#realtime-presence). Presence
 
 h3(#eventemitter). EventEmitter mixin / interface
 
-* @(RTE1)@ @EventEmitter@ is a generic interface for event registration and delivery used in a number of the types in the Realtime client library.  For example, the @Connection@ object emits events for connection state using the @EventEmitter@ pattern
+* @(RTE1)@ @EventEmitter@ is a generic interface for event registration and delivery used in a number of the types in the Realtime client library. For example, the @Connection@ object emits events for connection state using the @EventEmitter@ pattern
 * @(RTE2)@ Where objects provide @subscribe@ or @unsubscribe@ methods, they should follow the specification for the @EventEmitter#on@ and @EventEmitter#off@ methods respectively
-* @(RTE3)@ @EventEmitter#on@ registers a listener for the specified event if provided, else it is registered for all events
-* @(RTE4)@ @EventEmitter#once@ registers a listener that is called only once when the first of the applicable events occurs (ie the specified event if given, or the first event if it is registered for all events)
-* @(RTE5)@ @EventEmitter#off@ deregisters a listener for the specified event if given, else it is deregisters that listener for all registrations (global and event-specific)
-* @(RTE6)@ @EventEmitter#emit@ emits an event, calling registered listeners with the given event name any other given arguments. If an exception is raised in any of the listeners, the exception is caught by the @EventEmitter@ and the exception is logged to the Ably logger. Tests must exist to ensure exceptions raised in client code do not propagate and inhibit other event processing within the client library
+* @(RTE3)@ @EventEmitter#on@ registers the provided listener for either all events when no @event@ argument is provided, or for only a single type of event when an @event@ argument is provided. If @on@ is called more than once with the same listener, the listener is added multiple times to its listener registry. Therefore, as an example, assuming the same listener is registered twice using @on@, and an event is emitted once, the listener would be invoked twice
```

Comment:
Suggest "or for only a single type of event" -> "or for only a single named event"

"If @on@ is called more than once with the same listener" -> "If @on@ is called more than once with the same listener and event"